experimental/dds/tree2/src/feature-libraries/editable-tree/utilities.ts

```diff
@@ -118,3 +122,41 @@ export function treeStatusFromPath(path: UpPath): TreeStatus {
 		? TreeStatus.InDocument
 		: TreeStatus.Removed;
 }
+
+/**
+ * Checks the detached field and returns the TreeStatus based on whether or not the detached field is a root field.
+ * @param detachedField - the detached field you want to check.
+ * @returns the {@link TreeStatus} from the detached field provided.
+ */
+export function treeStatusFromDetachedField(detachedField: DetachedField) {
+	return detachedField === rootField ? TreeStatus.InDocument : TreeStatus.Removed;
+}
+
+export function treeStatusFromAnchorCache(
+	proxy: FieldProxyTarget | NodeProxyTarget,
```

Comment:
instead of taking a proxy, you could have this take an AnchorSet since that's all it needs. this makes the helpers more reusable and will make porting this logic to the new editable tree easier

Reply:
Updated to take in `AnchorSet`